README.md

```diff
@@ -34,7 +34,7 @@ The `main` branch is the bleeding edge of the v1 migration. The `master-v0.x` br
 v0 release (where we are still merging some patches and crucial fixes).
 
 The current v1 release is
-<a title="Pub" href="https://pub.dartlang.org/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>
+<a title="Pub" href="https://pub.dev/packages/flame/versions/1.0.0-releasecandidate.13" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>
 on pub. The latest stable version so far is
 <a title="Pub" href="https://pub.dartlang.org/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout" /></a>.
```

Comment:
Why isn't this URL updated? `pub.dev`?

Reply:
I made the PR to point to the direct version of the RC and update the broken forge2d link, the pub.dev update was a side effect :) I went ahead and updated all other references in [another commit](https://github.com/flame-engine/flame/pull/944/commits/7d165fb2e86080a7e64f429f9d0e703ca313d06d).

There were two more pages that didn't have the latest RC link (one pointed to the older version,) [here](https://github.com/flame-engine/flame/pull/944/commits/4c98d04b2296bf0eeece4632555266dacd84a55e).

As there are so many references, maybe they should all be updated to point to the prerelease tag. Unless there is somewhere in the repo that has the latest pre-release version number, a dynamic link could be built using something [like this](https://github.com/marketplace/actions/dynamic-readme) which will always point to the latest release.